iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/models/DeploymentServiceModel.java

```diff
@@ -3,17 +3,14 @@
 package com.microsoft.azure.iotsolutions.iothubmanager.services.models;
 
 import com.microsoft.azure.iotsolutions.iothubmanager.services.exceptions.InvalidInputException;
+import com.microsoft.azure.iotsolutions.iothubmanager.services.external.ConfigurationsHelper;
 import com.microsoft.azure.sdk.iot.service.Configuration;
 import org.apache.commons.lang3.StringUtils;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
 
 public class DeploymentServiceModel {
```

Comment:
Just an FYI this is the correct naming convention

Reply:
Well I had a discussion with Elvin. Lets talk on Monday.

Reply:
done.